src/v/serde/serde.h

```diff
@@ -46,14 +46,6 @@
 
 namespace serde {
 
-template<typename T>
-inline constexpr bool serde_is_enum_v =
```

Comment:
To make 7e7d5e2b2b920fcdddbe5a21d05429709e261bdf compile correctly you'll need to include `serde_is_enum.h` somewhere right?

Reply:
Added `#include "serde/serde_is_enum.h`.